features/employers/benefit_applications/admin_terminating_benefit_application_steps.rb

```diff
@@ -59,6 +59,7 @@
 
 And(/^employer clicks OK in warning modal$/) do
   # find('.swal-button swal-button--confirm swal-button--danger', text: "OK").click
+  find_button('OK').visible?
```

Comment:
we should not be asserting on hard-coded english text, instead let's add a `data-cuke` attribute to the button and assert on that